.github/workflows/ci.yml

```
@@ -12,7 +12,7 @@ on:
   push:
     branches: [main]
   pull_request:
-    branches: [main]
+    branches: [main, v2-1/dev, v2/uncertainty]
```

Remove this before merging
```suggestion
    branches: [main]
```